bindings/pydeck/pydeck/widget/_frontend.py

```diff
@@ -9,4 +9,4 @@
 module_name = "@deck.gl/jupyter-widget"
 # module_version is the current version of the module of the JS portion of the widget
 # It appears to be important only for JupyterLab and ignored for Jupyter Notebooks
-module_version = "^8.0.0"
+module_version = "8.1.0-alpha.1"
```

Comment:
Can we not hardcode any version?

Reply:
Handled in a separate PR, this is only for development purposes, I'll change it back